resources/js/visual-editor/editor/components/editor.css

```
@@ -19,7 +22,42 @@
     --ve-color-border-strong: #d0d0d0;
     --ve-color-dirty: #bf5af2;
     --ve-color-error: #bf1d1d;
+    --ve-color-accent: #0a84ff;
+
+    /* Typography — themeable */
     --ve-font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif;
```

Stylelint (17.7.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)